.github/workflows/ci.yml

```diff
@@ -0,0 +1,43 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        ruby: [ '2.6.3' ]
```

Comment:
2.6.5?

https://github.com/ruby/www.ruby-lang.org/blob/cfcb63e3264df8dba8dec7138b09e663ef540f40/.travis.yml#L3

Reply:
See above; 2.6.5 currently is not provided by the setup-ruby action.

Reply:
🤦‍♂  I see. Sorry.